pkg/registrationserver/routes.go

```diff
@@ -0,0 +1,66 @@
+//go:generate go run -tags=dev pkg/static/assets_generate.go
```

Comment:
This needs to be
```
//go:generate go run -tags=dev ../../pkg/static/assets_generate.go
```

Reply:
..it is always contextualized from the directory the directive is found. If you move `main.go` into `pkg`, this should also be moved back to `main.go` (and the path again updated).